README.md

```diff
@@ -13,17 +15,34 @@ For more information about contributing to the Kubernetes documentation, see:
 * [Using Page Templates](http://kubernetes.io/docs/contribute/style/page-templates/)
 * [Documentation Style Guide](http://kubernetes.io/docs/contribute/style/style-guide/)
 
+## Running the site locally using Hugo
+
+The Kubernetes documentation is built using the [Hugo](https://gohugo.io) static site generator. See the [official Hugo documentation](https://gohugo.io/getting-started/installing/) for Hugo installation instructions.
+
+> The site currently requires Hugo version **0.46**.
```

Comment:
I think you can set this as a Hugo variable in the and use that variable here. It stinks that we have to have it also in the README. Any suggestions, @bep ?

Reply:
This is a README.md meant to be rendered and read on GitHub. No variables available. But I would make the language less specific to make it simpler to maintain.